src/EFCore.PG/Extensions/NpgsqlBinaryStringExtensions.cs

```diff
@@ -0,0 +1,119 @@
+#region License
+
+// The PostgreSQL License
+//
+// Copyright (C) 2016 The Npgsql Development Team
```

Comment:
Back to the past? (:

Reply:
Nice catch... blindly copying and pasting from another file with aut-collapse turned on. Will update